Networking/Networking/Model/OrderCount.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+/// Represents an OrderCount Entity.
+/// An OrderCount contains an array of OrderCountItem.
+/// Each OrderCountItem representes the number of Orders for a given status
+///
+public struct OrderCount: Decodable {
+    public let siteID: Int
+    public let items: [OrderCountItem]
+
+    public init(siteID: Int, items: [OrderCountItem]) {
+        self.siteID = siteID
+        self.items = items
+    }
+
+    /// The public initializer for OrderCount.
+    ///
+    public init(from decoder: Decoder) throws {
+        guard let siteID = decoder.userInfo[.siteID] as? Int else {
+            throw OrderCountError.missingSiteID
+        }
+
+        let container = try decoder.singleValueContainer()
+        let items = try container.decode([OrderCountItem].self)
+
+        self.init(siteID: siteID, items: items)
+    }
+
+    /// Returns the first OrderCountItem mathing a order status slug
+    ///
+    public subscript(slug: String) -> OrderCountItem? {
+        get {
+            return items.filter {
+                $0.slug == slug
+            }.first
+        }
+        set {
```

Comment:
if the subscript is meant to be read-only (since the setter is no-op), can this be
```
public subscript(slug: String) -> OrderCountItem? {
  return items.filter {
      $0.slug == slug
  }.first
}
```

Reply:
Yes, it can! (TIL, although it makes total sense)

Reply:
That's one of the things I love about Swift, you learn something new (that in retrospect, makes total sense) everyday